viper/parser/parser_utils.py

```diff
@@ -165,6 +170,10 @@ def repr(self):
         o = ''
         if self.annotation:
             o += '/* %s */ \n' % self.annotation
+        if self.repr_show_gas and self.gas:
+            OKBLUE = '\033[94m'
+            ENDC = '\033[0m'
+            o += OKBLUE + "{" + ENDC + str(self.gas) + OKBLUE + "} " + ENDC  # add gas for info.
```

Comment:
Could you explain what this does?

Reply:
This just for the nice blue coloured braces one sees on the `--show-gas-estimates` output. I will also add a test for it ;)